main.py

```diff
@@ -0,0 +1,59 @@
+"""This module uses commands to call functions from counter.py."""
+import typer
+from castanet import generate_trees as generator
+from castanet import counter
+
+app = typer.Typer(help="Awesome CLI user manager.")
```

Comment:
This `help` string shows up as a description/help for your program as a whole when `main.py --help` is run. It may be more useful for users if a descriptive string about what the program does is used, as the current `help` is not related to CASTanet.